separate_curves.py

```diff
@@ -22,7 +22,7 @@
     "name": "Separate Curve Object",
     "author": "Agnieszka Pas",
     "version": (1, 1, 0),
```

Comment:
A version should be changed, lets say into (2, 0, 0) as it's a significant change for Blender 2.80